llvm/test/CodeGen/AMDGPU/GlobalISel/lrint.ll

```diff
@@ -0,0 +1,493 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -global-isel -mtriple=amdgcn -mcpu=gfx900 < %s | FileCheck -check-prefixes=GCN,GFX9 %s
+; RUN: llc -global-isel -mtriple=amdgcn -mcpu=gfx1010 < %s | FileCheck -check-prefixes=GCN,GFX10 %s
+; RUN: llc -global-isel -mtriple=amdgcn -mcpu=gfx1100 < %s | FileCheck -check-prefixes=GCN,GFX11 %s
```

Comment:
You can just add these run lines to the other copy of the test, we don't need to duplicate the GlobalISel tests anymore except in special cases. Here I would expect the output to be close enough to identical to easily share

Reply:
GlobalISel tests generate code using a scalar reg on bit operations where as selectiondag uses a vector. Does this still mean I should use a single test ?

Reply:
Yes. You can have common check prefixes and it will show where they differ